app/person-escort-record/controllers/framework-step.js

```diff
@@ -32,7 +32,14 @@ class FrameworkStepController extends FormWizardController {
 
   middlewareSetup() {
     super.middlewareSetup()
+    // TODO: Find a better way to fix the ordering of these middleware
```

Comment:
Maybe I'm getting the wrong end of the stick, but could you add a noop `setInitialValues` to the form-wizard controller and simply override that here?

Reply:
Ooo, that's a very good idea :)